### PR TITLE
Move maas

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* moved maas.html from /cloud to /server
* renamed it in the menu from MAAS to Server provisioning
* updated all the redirects
* updated the contact us in the contextual footer to the server on
* fixed up the _server.scss to inherit better and moved maas page styling there and out of _cloud.scss

## QA

1. go to /server/maas and see that it looks like the live /cloud/maas
2. see that the menu name is 'Server provisioning'
3. see that the contextual footer is now the server contact-us

## Issue / Card

[trello card](https://trello.com/c/40W4AhLa)

### CODE REVIEW

**Comment by anthonydillon** [comment]
QA issue:
The nav is not selected
maas server ubuntu

File: static/css/section/_server.scss [hunk 1]
```diff
@@ -29,90 +29,131 @@
       }
     }
   }
-}
 
-// @subsection /server/management
-.server-management {
+  // @subsection /server/management
+  &.server-management {
```
Comment (anthonydillon) [inline static/css/section/_server.scss]: :heart:

File: templates/server/_nav_secondary.html [hunk 2]
```diff
@@ -3,4 +3,5 @@
         <li{% if level_2 == 'hyperscale' %} class="last-item"{% endif %}><a{% if level_2 == 'management' %} class="active"{% endif %} href="/server/management">Server management</a></li>
         <li><a{% if level_2 == 'hyperscale' %} class="active"{% endif %} href="/server/hyperscale">Hyperscale</a></li>
         <li><a{% if level_2 == 'livepatch' %} class="active"{% endif %} href="/server/livepatch">Livepatch</a></li>
+        <li><a id="MAAS-menu-item" {% if level_2 == 'Server provisioning' %} class="active"{% endif %} href="/server/maas">Server provisioning</a></li>
```
Comment (anthonydillon) [inline templates/server/_nav_secondary.html]: Should be `{% if level_2 == 'maas' %}` as its based on the URL structure not the title.

**Comment by pmahnke** [comment]
@anthonydillon fixed... can you confirm?

**Pull request merged**